app/models/notice.rb

```diff
@@ -5,11 +5,11 @@ class Notice < ActiveRecord::Base
   # Paperclip attachment
   # The storage folder require the use of Sendfile.
   has_attached_file :image,
-                    :styles => {large: "400x400>",small: "250x250>"},
-                    :path => ":rails_root/storage/notices/:id/:style-:filename"
+                    styles: {large: "400x400>",small: "250x250>"},
```

Comment:
Space missing after comma.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.